vpr/src/base/read_options.cpp

```diff
@@ -1754,6 +1785,97 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("false")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    place_grp.add_argument(args.place_static_move_prob, "--place_static_move_prob")
+        .help(
+            "The Probability of different move in Simulated Annealing."
+            "The order of moves is: {uniformMove, MedianMove, CentroidMove, WeightedCentroid, WeightedMedian, Timing feasible Region(TFR), Critical UniformMove}")
+        .nargs('+')
+        .default_value({"100", "0", "0", "0", "0", "0", "0"})
+
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument(args.place_static_notiming_move_prob, "--place_static_notiming_move_prob")
+        .help(
+            "The Probability of different non timing move in Simulated Annealing."
+            "The order of moves is: {uniformMove, MedianMove, CentroidMove}")
+        .nargs('+')
+        .default_value({"100", "0", "0"})
+
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    place_grp.add_argument(args.place_high_fanout_net, "--place_high_fanout_net")
+        .help(
+            "Sets the assumed high fanout net during placement. ")
```

Comment:
What does that high fanout net threshold do?  Do you ignore nets with more than that fanout in certain directed moves?  Should say what this controls.

Reply:
Exactly, any net with higher fanout will be ignored in some directed moves. I have updated the help accordingly.